gax/src/main/java/com/google/api/gax/batching/Batcher.java

```diff
@@ -38,6 +38,8 @@
  * when triggered by an internal threshold. This is intended to be used for high throughput
  * scenarios at the cost of latency.
  *
+ * <p>This instance can only be used from a single thread.
```

Comment:
`Batcher instances are not thread safe. To use across different threads, create a new Batcher instance per thread`